doc/man3/EVP_PKEY_derive.pod

```diff
@@ -16,6 +17,9 @@ EVP_PKEY_derive_set_peer_ex, EVP_PKEY_derive_set_peer, EVP_PKEY_derive
                                  int validate_peer);
  int EVP_PKEY_derive_set_peer(EVP_PKEY_CTX *ctx, EVP_PKEY *peer);
  int EVP_PKEY_derive(EVP_PKEY_CTX *ctx, unsigned char *key, size_t *keylen);
+ EVP_SKEY *EVP_PKEY_derive_SKEY(EVP_PKEY_CTX *ctx, EVP_SKEYMGMT *mgmt,
```

Comment:
I might have needed a proper review of the design..
Is this API ok?
Should the EVP_SKEYMGMT maybe be a setter on the EVP_PKEY_CTX instead of doing this? (Or does that not make sense?)

Reply:
I think it will work. I don't see why it's needed for the EVP_PKEY_CTX as it is used only for EVP_SKEY